e2e/webpack/bundle-through/webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+
+module.exports = [
+	{
+		mode: 'production',
+		entry: './index.mjs',
+		target: 'node',
+		optimization: {
+			minimize: false,
+		},
+		output: {
+			path: path.resolve(__dirname, 'dist'),
+			filename: 'bundle-from-mjs.cjs',
+		},
+	},
+	{
+		mode: 'production',
+		entry: './index.cjs',
+		target: 'node',
+		optimization: {
+			minimize: false,
+		},
+		output: {
+			path: path.resolve(__dirname, 'dist'),
+			filename: 'bundle-from-cjs.cjs',
+		},
+	},
+];
```

Comment:
I am not a Webpack wizard but I managed to make it do two things :
- take `esm` and bundle
- take `cjs` and bundle

The output is always a commonjs bundle.
I have no idea how to make Webpack output `esm` and reading their docs sucks the life out of me :D

Reply:
🧙🏼‍♂️